### ISSUE TITLE
deluxetable vs siunitx

### ISSUE THREAD

**Issue by michaelaye** [AASJournals/AASTeX60#61]
Here's a nut to crack:

AASTex "officially" recommends the `deluxetable` package, however it clashes with `siunitx` which is a wonderful package to write clean and consistent scientific-notation numbers without constantly having to drop down to math-mode in your otherwise prosaic text, which looks very cluttered and bug-rich.

Are there in the mean time maybe better alternatives to `deluxetable`? Is there an established pre-amble that avoids these clashes (`tablenum` is defined in both)? I found a workaround [here](https://tex.stackexchange.com/questions/192610/use-emulateapj-aastex-with-siunitx) using the `savesym` package, but wondered if there are no better, more integrated solutions available?

As a scientific-focused Latex environment, I think AASTeX should also officially recommend the `siunitx` package, but then it would create the aforementioned clashes, hence my GH issue to start a discussion about this.

**Comment by augustfly** [comment]
@michaelaye not sure what to say. `deluxetable` has an extreme momentum problem. kind of like `bibtex`. ;-) 

we might consider this if we completely start over.

**Comment by michaelaye** [comment]
Consider what exactly? Not sure what you are suggesting?

Don't you think to have consistent scientific notation is worth the effort to investigate how the packages can be healthily combined? I understand the value of deluxetable, but was wondering if the other table packages have maybe now/these days not improved in ways that renders deluxetable obsolete? (Honest question, I don't know). 
For everyone to either write "20 $^\circ$C" all the time, or even looking up on how to write your own macro, it is just soo many human hours wasted compared to simply doing "\SI{20}{\celsius}, not even talking about readability.
 
But the problem is apparently one level lower, with `deluxetable` clashing with the package `array` that is being loaded by `siunitx`

**Comment by augustfly** [comment]
yeah, I understand, yet this will be labeled as a style discussion instead of bug to be fixed. if additional authors' opinions elevate this discussion then we can tag it for a more comprehensive solution, which would be in v6.3 at the earliest.

**Comment by kadrlica** [comment]
I vote to elevate this discussion. I just ran into this error trying to use `aastex61` + `siunitx`:
```
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!
! LaTeX error: "xparse/command-already-defined"
! 
! Command '\tablenum' already defined!
! 
! See the LaTeX3 documentation for further information.
! 
! For immediate help type H <return>.
!...............................................  
```

**Comment by michaelaye** [comment]
yeah, that is exactly the issue.

**Comment by michaelaye** [comment]
btw, i guess the repo really should be renamed to AASTEX ? ;)

**Comment by augustfly** [comment]
@gregschwarz you are tracking v6.2 -- revising `deluxetable` to deal with some of these issues is on/off the table?

**Comment by gregschwarz** [comment]
I have asked Amy to look into the siunitx and the other odd centering problem but she is currently on vacation. Hopefully there will be an easy fix when she gets back but I can't make any promises that these will be addressed before v6.2 is released

**Comment by devonhollowood** [comment]
Just ran into this issue today. Any updates now that v6.2 is out?

**Comment by devonhollowood** [comment]
Just ran into this issue today. Any updates now that v6.2 is out?

**Comment by syrte** [comment]
@devonhollowood This link may help https://tex.stackexchange.com/a/269074

**Comment by augustfly** [comment]
We tried to manage this conflict with a work around added during v6.3 development. The documentation for this work around is here:

https://journals.aas.org/aastexguide/#conflicts

replies requested.
